server_environment/static/description/index.html

```diff
@@ -543,11 +543,11 @@ <h1><a class="toc-backref" href="#toc-entry-8">Known issues / Roadmap</a></h1>
 <ul class="simple">
 <li>it is not possible to set the environment from the command line. A
 configuration file must be used.</li>
-<li>the module does not allow to set low level attributes such as database
-server, etc.</li>
+<li>the module does not allow to set low level attributes such as
+database server, etc.</li>
```

Comment:
FYI if you want to disable readme update when not needed, you can do `export SKIP=oca-gen-addon-readme` :wink:

Reply:
Thanks for the tip.